src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
```

Comment:
```suggestion
```
`scheduled` doesn't work anymore. After I approve, please merge this PR when you want it to go live. We deploy the site every hour, and the deploy process takes about 20 minutes.